microcloud/test/suites/add.sh

```diff
@@ -99,7 +99,9 @@ test_add_auto() {
 }
 
 test_add_interactive() {
-  reset_systems 4 2 1
+  ceph_public_subnet="10.0.1.0/24"
+  ceph_cluster_subnet="10.0.2.0/24"
+  reset_systems 4 2 4 "${ceph_public_subnet}" "${ceph_cluster_subnet}"
```

Comment:
By default, each machine is only created expecting a maximum of 3 disks and 3 interfaces. If you want to increment this higher (like 4, which you have here), you would have raise the values passed [here](https://github.com/canonical/microcloud/blob/0c6fff36559abb2277e4dcb08d146992a31b8400/microcloud/test/main.sh#L128) to `new_systems 4 3 4`

Reply:
fixed